pennylane/gradients/general_shift_rules.py

```diff
@@ -20,20 +22,26 @@
 import pennylane as qml
 
 
-def process_shifts(rule, tol=1e-10, check_duplicates=True):
+def process_shifts(rule, tol=1e-10, batch_duplicates=True):
     """Utility function to process gradient rules.
 
     Args:
-        rule (array): a ``(N, M)`` array corresponding to ``M`` terms
+        rule (array): a ``(M, N)`` array corresponding to ``M`` terms

```

Comment:
oh wait, was this a bug in the docstring, or the behaviour of the function has changed?

Reply:
The behaviour was changed :)